.github/workflows/ci.yml

```diff
@@ -36,3 +36,14 @@ jobs:
 
       - name: Build
         run: npm run build
+
+  release:
+    build:
+      runs-on: ubuntu-latest
+      needs:
+      if: startsWith(github.ref, "refs/tags/")
```

Comment:
I am not sure how it works.. on tags push ?

Reply:
Yes.